lib/osf-components/addon/components/file-browser/template.hbs

```diff
@@ -3,32 +3,33 @@
 <div local-class='OptionBar {{if this.isMobile 'Mobile'}}'>
     <div local-class='OptionBar__left {{if this.isMobile 'Mobile'}}'>
         <Input
+            aria-label={{t 'registries.overview.files.filter_label'}}
             data-test-file-search
             @type='text'
             class='form-control'
             placeholder='{{t 'general.filter'}}'
             @value={{readonly @manager.filter}}
-            {{on 'keyup' this.handleInput}}
+            {{on 'input' this.handleInput}}
```

Comment:
Is this behaving properly, or is it missing the last input still? Easiest to see when you clear out the contents of the filter.

Reply:
Yea, this one works as expected! Clearing the filter textbox will re-fetch all the files, typing in `Until` will return no results when I only have files called `UntitledX`, etc